.github/workflows/ubuntu-oldclang-18-ci.yml

```
@@ -0,0 +1,23 @@
+name: Ubuntu-CI (old llvm)
+
+on: push
+
+
+jobs:
+  ci:
+    name: ubuntu-clangold-gcc
+    runs-on: ubuntu-18.04
+
+    env:
+      CC: clang
+      CXX: clang++
+
+    steps: 
+      - uses: actions/checkout@v2
```

I choose clang 7.
See https://packages.ubuntu.com/search?keywords=clang for all clang related packages.
```suggestion
    env:
      CC: clang-7
      CXX: clang++-7

    steps: 
      - uses: actions/checkout@v2
      - name: install clang 7
        run: |
          sudo apt update
          sudo apt install clang-7
```